content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,160 @@
+---
+title: April '20 DVC❤️Heartbeat
+date: 2020-04-06
+description: |
+  DVC discussions around the web,
+  our growing team, and recommended reading from the open source community.
+descriptionLong: |
+  Every month we share news, findings, interesting reads, community takeaways,
+  and everything else along the way.
+
+  Look here for updates about [DVC](https://dvc.org), our journey as a startup,
+  projects by our users and big ideas about best practices in ML and data
+  science.
+picture: ../../static/uploads/images/2020-04-06/april_header.png
+pictureComment:
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/march-20-heartbeat/335
+tags:
+  - Heartbeat
+  - Google Drive
+  - MLOps
+  - CI/CD
+  - Podcast
+  - DivOps
+---
+
+Welcome to the April Heartbeat, our monthly roundup of cool happenings, good
```

Comment:
minor: make a link here - https://dvc-landing-april-heart-6d0onb.herokuapp.com/tags/heartbeat

Reply:
got it, let me know if the link is in the right place.